src/peakshapes.jl

```diff
@@ -250,3 +250,25 @@ function double_gaussian(
 end
 export double_gaussian
 
+
+###############################################################
+# additional peakshape functions for two EMG background
+###############################################################
+
+function two_emg_aoe_compton_peakshape( # total fit function with two EMGs
+    x::Real, 
+    μ::Real, σ::Real, n::Real, background::Real, δ::Real, 
+    μ2::Real, σ2::Real, background2::Real, δ2::Real
+)
+    return n * gauss_pdf(x, μ, σ) + background * ex_gauss_pdf(-x, -μ, σ, δ) + background2 * ex_gauss_pdf(-x, -μ2, σ2, δ2)
```

Comment:
Let's catch also here the case for `sigma` = 0.0 similar to the energy peakshape. Otherwise we might run again into numerical instabilites.